1 table.sql

```diff
@@ -0,0 +1,30 @@
+use MusicNoteDB 
+go 
+
+drop table if exists MusicRental
+go 
+create table dbo.MusicRental(
+    InstrumentId int not null identity primary key,  
```

Comment:
The pk should be the name of the table. Either name your table Instrument or your pk MusicRentalId

Reply:
Primary key renamed MusicRental